tests/draft2020-12/dynamicRef.json

```diff
@@ -726,5 +726,32 @@
                 "valid": false
             }
         ]
+    },
+    {
+        "description": "$dynamicRef points to a boolean schema",
+        "schema": {
+            "$defs": {
+                "boolean": true
+            },
+            "type": "object",
```

Comment:
You can drop the `type` here at least, and the test should go to `draft-next` too. Otherwise seems OK to me.

Reply:
done.